src/js/view/controls/controlbar.js

```diff
@@ -113,7 +113,8 @@ export default class Controlbar {
 
             const volumeButtonEl = volumeGroup.element();
             menus.push(volumeGroup);
-            setAttribute(volumeButtonEl, 'role', 'button');
+            // Use 'group' role so that voiceover identifies volume slider and button
+            setAttribute(volumeButtonEl, 'role', 'group');
             _model.change('mute', (model, muted) => {
```

Comment:
Because we have the volume slider as a child of the volume button, the only way to get it to be detected is to treat the button as a group.
I think the only way to completely solve this would be to refactor this and separate the button and the slider into two elements. This is the workaround that others who faced similar issues have used as well.